docs/testing-guidelines/WritingPesterTests.md

```diff
@@ -69,81 +55,91 @@ Note that if `Get-Item` were to succeed, the test will fail.
 
 However, if you need to check the `InnerException` or other members of the ErrorRecord, you should use `-PassThru` parameter:
 
-```powershell
-	It "InnerException sample" {
-
-	$e = { Invoke-WebRequest https://expired.badssl.com/ } | Should -Throw -ErrorId "WebCmdletWebResponseException,Microsoft.PowerShell.Commands.InvokeWebRequestCommand" -PassThru
-	$e.Exception.InnerException.NativeErrorCode | Should -Be 12175
-    ...
-	}
+```PowerShell
+It "InnerException sample" {
+   $e = { Invoke-WebRequest https://expired.badssl.com/ } | Should -Throw -ErrorId "WebCmdletWebResponseException,Microsoft.PowerShell.Commands.InvokeWebRequestCommand" -PassThru
+   $e.Exception.InnerException.NativeErrorCode | Should -Be 12175
+}
 ```
 
-### Describe/Context/It
+## Describe/Context/It
+
+For creation of PowerShell tests, the `Describe` block is the level of granularity suggested and one of three tags should be used: `CI`, `Feature`, or `Scenario`.
 
-For creation of PowerShell tests, the `Describe` block is the level of granularity suggested and one of three tags should be used: "CI", "Feature", or "Scenario". If the tag is not provided, tests in that describe block will be run any time tests are executed.
+If the tag is not provided, tests in that describe block will be run any time tests are executed.
```

Comment:
I think we don't run them at all now, and there's an error during the build that the tag is missing.

Reply:
Very good catch @JamesWTruher 👍. Unfortunately the Linux CI build doesn't fail. See issue #9193 for more information.